managed/ui/src/components/tables/ListBackups/ListBackups.js

```diff
@@ -408,7 +408,7 @@ export default class ListBackups extends Component {
                 {isAvailable(currentCustomer.data.features, 'universes.backup') && (
                   <div className="backup-action-btn-group">
                     <TableAction
-                      disabled={currentUniverse.universeDetails.backupInProgress}
+                      disabled={currentUniverse.universeDetails.backupInProgress || currentUniverse.universeConfig['takeBackups'] === "false" ? true : false}
```

Comment:
- You can access an Object property with a simple object accessor like so `currentUniverse.universeConfig.takeBackups`. Use brackets only if passing a variable or number.
- Don't need the ternary to set to `true` or `false`, it is implied.
- Make sure you check that `currentUniverse.universeConfig` is true before trying to access `takeBackups` property.

Reply:
I think a good idea would be to change `takeBackups` to boolean type, not sure why it's a string

Reply:
@yb-andrew regarding second point using ternary operator, As ```takeBackups``` is of type string. I will address other points and raise the PR